lib/ansible/modules/network/ios/ios_l2_interface.py

```diff
@@ -146,7 +146,7 @@ def is_switchport(name, module):
 
 def interface_is_portchannel(name, module):
     if get_interface_type(name) == 'ethernet':
-        config = get_config(module, flags=[' | section interface'])
+        config = run_commands(module, ['show run interface {0}'.format(name)])[0]
         if 'channel group' in config:
```

Comment:
It looks like this should be added possibly or something similar?
```python
        if 'channel group' in config:
            return True
        elif 'channel-group' in config:
            return True
```

Reply:
@mrlesmithjr Thanks for bringing up the other scenario. The latest commit should fix that.

Reply:
Works as it should! Great job!